ControlRoomApplication/ControlRoomApplication/Entities/WeatherStation/WeatherStation.cs

```diff
@@ -103,7 +102,8 @@ protected void InitializeStation(int commPort)
         {
             if (OpenCommPort_V((short)commPort, 19200) != 0)
             {
-                logger.Error(Utilities.GetTimeStamp() + ": OpenCommPort unsuccessful!");
+                    logger.Error(Utilities.GetTimeStamp() + ": OpenCommPort unsuccessful!");
```

Comment:
I think there should only be one tab of indentation instead of two. I see that the original was one tab, but you added another to line up with line 107. Looking at the surrounding code, it seems that it all uses 1 tab of indentation. This indentation change should probably be reverted, and line 107 should be reduced to 1 tab of indentation.

- [ ] Consider changing to 1 tab of indentation